test/FateRewardControllerV3/migrate.js

```diff
@@ -84,6 +84,8 @@ describe('FateRewardControllerV3.migrate', () => {
             const length = await fateRewardControllerV3.poolLength()
             await fateRewardControllerV3.setMigrator(this.bob.address)
             // await fateRewardControllerV3.connect(this.bob).migrate(testLP1.address)
+            // TODO fix: this line causes a revert because there is no poolId at length. The last one is at length - 1.
+            //  But you must check length > 0!
             const lastLP1 = await fateRewardControllerV3.poolInfo(length)
             expect(lastLP1.lpToken).to.be.equal(testLP1.address)
```

Comment:
This test should be fixed. See my comment for context. Uncomment line `86`, ensuring `migrate` works.